### ISSUE TITLE
Feature request: check_perfmon.exe - Change name of counter in output

### ISSUE THREAD

**Issue by Stefar77** [Icinga/icinga2#5418]
When you create a check using check_perfmon.exe you can't change the name of the counter in the output / perfdata. 

## Current Behavior

check_perfmon.exe -P "\PhysicalDisk(_Total)\Current Disk Queue Length"
```
PERFMON OK "\PhysicalDisk(_Total)\Current Disk Queue Length" = 0
```

## Possible Solution

Add a parameter like -N {Name} that if set will override the name in the Output and perfdata
check_perfmon.exe -P "\PhysicalDisk(_Total)\Current Disk Queue Length" -N "Disk Queue Length"
While at it maybe format the output a bit more like below. ;-)
```
PERFMON OK - Disk Queue Length: 0
```

This will also be less of a pain when making graphs.

**Comment by mcktr** [comment]
For the perdata there is already a parameter 
```
--perf-syntax arg                Use this string as name for the performance
                                 counter (graphite compatibility)
``` 
As mentioned this will only override the counter name in the perfdata. I don't know if it useful to change the name of the counter in the output because then you don't have any clue where those data come from (by only looking on the output).

**Comment by Stefar77** [comment]
@mcktr You name the service so you know what it is, otherwise I need to rewrite it in our dashboard but you still see it when you get an alert from it, look in Icinga GUI or aNag. 
I changed it in a patch in my repo but I need to compile stuff on Windows but I really hate Windows. :-)

**Comment by mcktr** [comment]
I get your point, but won't it be useful to use the existing parameter `--perf-syntax` for the rewrite? With adding a new parameter like `-N` we have two parameters that will do almost the same.

**Comment by Stefar77** [comment]
True, in my case I would only need one parameter for all.
Service usually has a good name already, the output can say anything pointing people more in the right direction to know the issue.

Maybe other people would like the original counter name with " etc but I doubt it, if needed you can wrap it in your script but then you could also rewrite anything.. :-)

:+1:

**Comment by Crunsher** [comment]
I don't think that makes sense for check_perfmon.
As @mcktr said, there is already `--perf-syntax`. And other than a prettier display, there is no reason for the user to change the non-performance output.
An additional parameter is out of the question (dosen't add enough) and changing `--perf-syntax` is an unexpected change I'd rather not have.

What's the issue with a wrapper script/changing it in the dashboard?

**Comment by Stefar77** [comment]
Here, it's just looking ugly on any dashboard.
I'll fix a Windows host to compile stuff on and I'll just use my own patched version of the plugin. :-1: 

![noc](https://user-images.githubusercontent.com/28925335/28329470-3545b6a8-6bea-11e7-9722-fd206a46aa1a.png)

**Comment by Crunsher** [comment]
@N-o-X This is something for you. Don't forget to update the ITL and Documentation

**Comment by Crunsher** [comment]
Closed with #d1d1c2e
